src/test/java/com/puppycrawl/tools/checkstyle/utils/AnnotationUtilTest.java

```diff
@@ -226,7 +226,7 @@ public void testContainsAnnotationListWithNoAnnotationNode() {
     }
 
     @Test
-    public void testContainsAnnotationListWithEmptyAnnotationNode() {
+    public void testContainsAnnotationListWithIncompleteAnnotationNode() {
```

Comment:
>testContainsAnnotationListWithIncompleteAnnotationNode

This is good example of why pure Unit testing is bad. Operations on incomplete ast.

Is it complicated to remove this test  and cover required code by Input ?

Reply:
The code is already covered in the 4 tests I wrote. I will just remove this one.